src/css/header.css

```diff
@@ -0,0 +1,474 @@
+:root {
```

Comment:
remove all duplicating styles
variables are defined globally
use `container` and `section` global classes